lib/lotus/model/adapters/sql/mysql_console.rb

```diff
@@ -0,0 +1,49 @@
+module Lotus
+  module Model
+    module Adapters
+      module Sql
+        class MysqlConsole
+          def initialize(uri, options = {})
+            @uri = uri
+            @options = options
+          end
+
+          def connection_string
+            str = "mysql -h #{host} -D #{database}"
+            str << port if port
+            str << username if username
+            str << password if password
+            str
+          end
+
+          private
+
+          def host
+            @options.fetch('host') { @uri.host }
+          end
+
+          def database
+            database = @options.fetch('database') { @uri.path }
```

Comment:
It's worth to parse it through `ShellWord` because URI tend to be full file path